src/cli/preview.js

```diff
@@ -9,7 +9,22 @@ marked.setOptions({
   renderer: new TerminalRenderer(),
 });
 
-module.exports.preview = function (inputs) {
+module.exports.preview = function (inputs, flags) {
+  if (!!flags && flags.previousVersion) {
+    const previousVersionRegex = new RegExp(
+      `# \\[${flags.previousVersion}\\] - \\(.*\\)([^\\[]+)\\.`
```

Comment:
🤔 if someone change the default template, will this feature work? 

I think this feature should work for any scenario and I think it will not be that simple.

Reply:
In order of our talk, I've added a metadata injection in between the generated changelog so we don't need to worry about custom templates. Nice idea, though 🥇 . What do you think now?